src/Components/test/E2ETest/scripts/sauce.ts

```diff
@@ -0,0 +1,82 @@
+import { EOL } from "os";
```

Comment:
Why is this a typescript file? There's nothing typescript specific here and we shouldn't need typescript for something as simple as this.

Can we change this to be a plain JavaScript file? Any recent version of node will handle modern javascript.

Reply:
I took the easy path as we do similar things in [SignalR](https://github.com/dotnet/aspnetcore/blob/master/src/SignalR/clients/ts/FunctionalTests/scripts/run-tests.ts) which is in typescript. IMO there is no downside to using typescript for this. It is nicer :)